pageserver/src/disk_usage_eviction_task.rs

```diff
@@ -523,7 +518,11 @@ async fn extend_lru_candidates(
                 .into_iter()
                 .map(|layer_infos| (tl.clone(), layer_infos)),
         );
-        max_layer_size = max_layer_size.max(info.max_layer_size.accurate());
+        max_layer_size = match (max_layer_size, info.max_layer_size) {
+            (Some(x), Some(y)) => Some(x.max(y)),
+            (Some(only), None) | (None, Some(only)) => Some(only),
+            (None, None) => None,
+        };
```

Comment:
> However I do not like to use the max like this because it doesn't work with min due to how Option variants have been declared.

Can you elaborate?

Reply:
Went without option